src/test_l1_dsFPD.c

```diff
@@ -1041,10 +1007,7 @@ void test_l1_dsFPD_positive_dsGetFPState (void)
         UT_ASSERT_EQUAL(result, dsERR_NONE);
     }
 
```

Comment:
do we need compare current state by calling again getfpstate like other test cases ?

Reply:
updated